.readme-partials.yaml

```diff
@@ -46,6 +48,106 @@ custom_content: |
    - oauthToken (string): A valid pre-existing OAuth token to use for authentication for this connection. Setting this property will take precedence over any value set for a credentials file.
    - lenient (boolean): Enable this to force the JDBC driver to ignore unknown properties in the connection URL. Some applications automatically add additional properties to the URL that are not recognized by the JDBC driver. Normally, the JDBC driver will reject this, unless `lenient` mode is enabled.
    
+   ### OpenTelemetry Tracing
+   The Spanner JDBC driver supports OpenTelemetry tracing. You can configure the OpenTelemetry instance
+   that should be used in two ways:
+   1. Register a global OpenTelemetry instance. This instance will automatically be picked up by the Spanner JDBC driver.
+   2. Add an OpenTelemetry instance with the key `openTelemetry` to the `java.util.Properties` instance that is used to create the JDBC connection.
+   
+   By default, the traces that are generated by the Spanner JDBC driver do not include the SQL
+   statement. You can include the SQL statement with the traces by adding the property `enableExtendedTracing=true`
+   to the JDBC connection URL.
+   
+   #### Example Using Global OpenTelemetry
+   Create and register a global OpenTelemetry object before creating a JDBC connection.
+   See also the [Spring Data JDBC Sample](samples/spring-data-jdbc) for an example for how to
+   configure OpenTelemetry in combination with Spring Data.
+   
+   See [Latency Debugging Guide](documentation/latency-debugging-guide.md) for more information on how to use these traces.
+
+   ```java
+   TraceConfiguration traceConfiguration = TraceConfiguration.builder().setProjectId("my-project").build();
+   SpanExporter traceExporter = TraceExporter.createWithConfiguration(traceConfiguration);
+   OpenTelemetry openTelemetry =
+        OpenTelemetrySdk.builder()
+            .setTracerProvider(
+                SdkTracerProvider.builder()
+                    .setSampler(Sampler.traceIdRatioBased(0.05))
+                    .setResource(
+                        Resource.builder()
+                            .put("service.name", "my-unique-service-name")
+                            .build())
+                    .addSpanProcessor(BatchSpanProcessor.builder(traceExporter).build())
+                    .build())
+            .buildAndRegisterGlobal();
+   String projectId = "my-project";
+   String instanceId = "my-instance";
+   String databaseId = "my-database";
+   // Setting this to true instructs the JDBC driver to include the SQL statement with the traces.
+   boolean enableExtendedTracing = true;
```

Comment:
Should we also `enableApiTracing` in the code snipper

Reply:
Added (although we need to wait for https://github.com/googleapis/java-spanner/pull/3166 to be merged before we can use it here)